wellregistry/wellregistry/custom_social_pipeline.py

```diff
@@ -0,0 +1,41 @@
+'''
+Custom social pipeline functions
+'''
+
+from django.contrib.auth.models import Group
+from django.conf import settings
+
+# pylint: disable=unused-argument
+
+def change_usgs_user_to_staff(strategy, details, backend, *args, user=None, **kwargs):
+    """
+    Social auth pipeline function to change usgs users automatically to have is_staff set and
+    to add the usgs permissions to the user.
+    :return: dict
+    """
+    if kwargs.get('is_new'):
+        email = details.get('username')
+        if email.find('@usgs.gov') != -1:
```

Comment:
Not a huge deal, but (subjectively) I think that that `'@usgs.gov' in email` is more. Additionally, using `in` is faster.

Reply:
Agreed. I also am adding "@contractor.usgs.gov" here to check for.